Sources/FuzzilliCli/Profiles/V8CommonProfile.swift

```diff
@@ -0,0 +1,570 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Fuzzilli
+
+public extension ILType {
+    static let jsD8 = ILType.object(ofGroup: "D8", withProperties: ["test"], withMethods: [])
+
+    static let jsD8Test = ILType.object(ofGroup: "D8Test", withProperties: ["FastCAPI"], withMethods: [])
+
+    static let jsD8FastCAPI = ILType.object(ofGroup: "D8FastCAPI", withProperties: [], withMethods: ["throw_no_fallback", "add_32bit_int"])
+
+    static let jsD8FastCAPIConstructor = ILType.constructor([] => .jsD8FastCAPI)
+
+    static let gcTypeEnum = ILType.enumeration(ofName: "gcType", withValues: ["minor", "major"])
+    static let gcExecutionEnum = ILType.enumeration(ofName: "gcExecution", withValues: ["async", "sync"])
+}
+
+public let gcOptions = ObjectGroup(
+    name: "GCOptions",
+    instanceType: .object(ofGroup: "GCOptions", withProperties: ["type", "execution"], withMethods: []),
+    properties: ["type": .gcTypeEnum,
+                 "execution": .gcExecutionEnum],
+    methods: [:])
+
+public let fastCallables : [(group: ILType, method: String)] = [
+    (group: .jsD8FastCAPI, method: "throw_no_fallback"),
+    (group: .jsD8FastCAPI, method: "add_32bit_int"),
+]
+
+// Insert random GC calls throughout our code.
+public let V8GcGenerator = CodeGenerator("GcGenerator") { b in
+    let gc = b.createNamedVariable(forBuiltin: "gc")
+
+    // `gc()` takes a `type` parameter. If the value is 'async', gc() returns a
+    // Promise. We currently do not really handle this other than typing the
+    // return of gc to .undefined | .jsPromise. One could either chain a .then
+    // or create two wrapper functions that are differently typed such that
+    // fuzzilli always knows what the type of the return value is.
+    b.callFunction(gc, withArgs: b.findOrGenerateArguments(forSignature: b.fuzzer.environment.type(ofBuiltin: "gc").signature!)) 
+}
+
+// Insert random GC calls throughout our code.
+public let V8GcSandboxGenerator = CodeGenerator("GcGenerator") { b in
```

Comment:
I think we can drop this one in favor of the above. I don't think we intended to have two different versions of this generator